packages/flutter/lib/src/services/text_input.dart

```diff
@@ -413,6 +417,40 @@ class TextInputConfiguration {
   /// Defaults to true.
   final bool autocorrect;
 
+  /// {@template flutter.services.textInput.enableSmartDashes}
+  /// Whether to allow the platform to automatically format dashes.
```

Comment:
It may be useful to explicitly explain what this is doing other than this very short. How is it being formatted?